rst_files/short_path.rst

```diff
@@ -158,7 +158,65 @@ Setup
+    graph = Dict(zip(0:99, [[(14, 72.21), (8, 11.11), (1, 0.04)],[(13, 64.94), (6, 20.59), 
```

Comment:
I think this is the one exception for the scrolling.  Can we revert?  Ultimately in #228 we will want to put this in an external file on the internet to load, but right now I think it looks better if it doesn't take up pages of text.